lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -636,8 +622,13 @@ MITIGATION_02_CRI_DCMAW:
     criId: dcmaw
   parent: CRI_STATE
   events:
+    next:
+      targetState: POST_DCMAW_SUCCESS_PAGE_J5
```

Comment:
Think we should remove this - the next event isn't (shouldn't be) actually emitted in any scenario here (in the 'success' case we'd match a profile and emit `end`)